lib/cadet/assessments/answer.ex

```diff
@@ -40,6 +40,14 @@ defmodule Cadet.Assessments.Answer do
     |> validate_xp_grade_adjustment_total()
   end
 
+  def reset(answer, params) do
```

Comment:
Since this returns a changeset, you might want to call this `reset_changeset` instead

Reply:
You should do `validate_required` as well. I think it's better to refactor the common part of `changeset/2` and `reset_changeset/2` into a private function instead.

Reply:
`validate_required` seems to need to preload the question in. Can we just assume since data has been added that it has already passed the validation initially? (we are not introducing new data) thats the reason why `reset_changeset` was created instead of just using changeset.

Reply:
I don't think that's a good idea. Why not just use the original `changeset/2`?